src/App.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from "react";
+import Sidebar from "./components/Sidebar";
+import Editor from "./components/Editor";
+import Split from "react-split";
+import { nanoid } from "nanoid";
+
+const App = () => {
+  const noteFromLocalStorage = localStorage.getItem("notes");
+  const [notes, setNotes] = useState(
+    () => JSON.parse(noteFromLocalStorage) || []
+  );
+
+  const [currentNoteId, setCurrentNoteId] = useState(
+    (notes[0] && notes[0].id) || ""
+  );
+
+  useEffect(() => {
+    localStorage.setItem("notes", JSON.stringify(notes));
+  }, [notes]);
+
+  const createNewNote = () => {
+    const newNote = {
+      id: nanoid(),
+      body: "# Type your markdown note's title here",
+    };
+
+    setNotes((prevNotes) => [newNote, ...prevNotes]);
+    setCurrentNoteId(newNote.id);
+  };
+
+  const updateNote = (text) => {
+    setNotes((oldNotes) => {
+      const updatedNoteArr = [];
+      for (let i = 0; i < oldNotes.length; i++) {
+        const oldNote = oldNotes[i];
+        if (oldNote.id === currentNoteId) {
+          updatedNoteArr.unshift({ ...oldNote, body: text });
+        } else {
+          updatedNoteArr.push(oldNote);
+        }
+      }
+      return updatedNoteArr;
+    });
+  };
+
+  const deleteNote = (event, noteId) => {
+    event.stopPropagation();
+    setNotes((oldNotes) => oldNotes.filter((note) => note.id !== noteId));
+  };
+
+  const findCurrentNote = () => {
+    return (
+      notes.find((note) => {
+        return note.id === currentNoteId;
+      }) || notes[0]
+    );
+  };
+
+  return (
+    <main>
+      {notes.length > 0 ? (
```

Comment:
you can use this short hand for checking the length truthy```suggestion
      notes.length > 0
```